test/session.test.js

```diff
@@ -597,11 +597,10 @@ describe('server session', function () {
     const sessions = await this.application.getSessions();
     const app2session = sessions.find(session => !session.current);
     await this.application.revokeSession(app2session.uuid);
-
-    const note = await Factory.createSyncedNote(app2);
-    await this.application.sync();
-    expect(this.application.findItem(note.uuid)).to.not.be.ok
-    app2.deinit();
+    void app2.sync();
+    /** Wait for app2 to deinit */
+    await Factory.sleep(3);
+    expect(app2.dealloced).to.be.true;
```

Comment:
It seems we’re testing that the app has been successfully destroyed, but not whether local data has been deleted. Because an app can be dealloced and still hang on to data on storage. So this test isn’t fully comprehensive I think.